tools/generators/epic-generator/index.ts

```diff
@@ -0,0 +1,191 @@
+import { getProjects, stripIndents, Tree } from '@nrwl/devkit';
+import { readFileSync } from 'fs';
+import { execSync } from 'child_process';
+import { EpicGeneratorSchema } from './schema';
+import { isPackageConverged } from '../../utils';
+
+function validateSchema(schema: EpicGeneratorSchema): Required<EpicGeneratorSchema> {
+  if (schema.repository !== undefined && !schema.repository.match(/[A-z-]+\/[A-z-]+/)) {
+    throw new Error('Must provide a valid repository with the format {owner}/{repositoryName}');
+  }
+
+  if ((schema.title || '').trim() === '') {
+    throw new Error('Must provide a title for the issue');
+  }
+
+  return {
+    repository: schema.repository || 'microsoft/fluentui',
```

Comment:
we can use schema.json default values for this , any reason why you decided to go this way ?

Reply:
Only because it could be tested, but I've migrated to schema defaults.